--- /home/ckl/3py/3proxy/scripts/3proxy.cfg	2009-07-14 16:17:31.000000000 +0800
+++ /home/ckl/3py/3proxy/scripts/3proxy_openwrt.cfg	2014-09-04 10:45:34.636403996 +0800
@@ -1,23 +1,23 @@
-#!/usr/local/etc/3proxy/bin/3proxy
+#!/usr/bin/3proxy
 daemon
-pidfile /usr/local/etc/3proxy/3proxy.pid
+pidfile /etc/3proxy/3proxy.pid
 nscache 65536
 nserver 127.0.0.1
 
-config /usr/local/etc/3proxy/3proxy.cfg
-monitor /usr/local/etc/3proxy/3proxy.cfg
-monitor /usr/local/etc/3proxy/counters
-monitor /usr/local/etc/3proxy/passwd
-monitor /usr/local/etc/3proxy/bandlimiters
+config /etc/3proxy/3proxy.cfg
+monitor /etc/3proxy/3proxy.cfg
+monitor /etc/3proxy/counters
+monitor /etc/3proxy/passwd
+monitor /etc/3proxy/bandlimiters
 
-log /usr/local/etc/3proxy/log/log D
+log /var/log/3proxy.log D
 rotate 60
-counter /usr/local/etc/3proxy/3proxy.3cf
+counter /etc/3proxy/3proxy.3cf
 
-users $/usr/local/etc/3proxy/passwd
+users $/etc/3proxy/passwd
 
-include /usr/local/etc/3proxy/counters
-include /usr/local/etc/3proxy/bandlimiters
+include /etc/3proxy/counters
+include /etc/3proxy/bandlimiters
 
 auth strong
 deny * * 127.0.0.1
