Index: 3proxy/Makefile.Linux
===================================================================
--- 3proxy.orig/Makefile.Linux	2014-09-04 14:35:24.920773288 +0800
+++ 3proxy/Makefile.Linux	2014-09-04 14:58:08.808809812 +0800
@@ -9,17 +9,17 @@
 # library support. Add -DSAFESQL for poorely written ODBC library / drivers.
 
 BUILDDIR =
-CC = gcc
 
-CFLAGS = -Wall -g -O2 -c -pthread -DGETHOSTBYNAME_R -D_THREAD_SAFE -D_REENTRANT -DNOODBC -DWITH_STD_MALLOC -DFD_SETSIZE=4096 -DWITH_POLL
+
+CFLAGS += -c -pthread -DGETHOSTBYNAME_R -D_THREAD_SAFE -D_REENTRANT -DNOODBC -DWITH_STD_MALLOC -DFD_SETSIZE=4096 -DWITH_POLL
 COUT = -o 
-LN = gcc
+LN = $(CC)
 DCFLAGS = -fpic
-LDFLAGS = -Wall -O2 -pthread
+
 DLFLAGS = -shared
 DLSUFFICS = .ld.so
 # -lpthreads may be reuqired on some platforms instead of -pthreads
-LIBS = -lcrypto -lssl -ldl 
+LIBS = -lcrypto -lssl -ldl -pthread 
 LNOUT = -o 
 EXESUFFICS =
 OBJSUFFICS = .o
